src/actions/editProjectAction.js

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios'
+import { EDIT_PROJECT_FAILURE } from '../types'
+
+export let editProject = props => dispatch => {
+  const { id, title, description, status, cookies } = props
+  return axios({
+    method: 'PUT',
+    url: `/projects/${id}`,
+    data: {
+      project: {
+        title,
+        description,
+        status
+      }
+    },
+    headers: {
+      Authorization: cookies.get(process.env.SESSION || 'WebsiteOne_session'),
+      Accept: 'application/json'
+    }
+  })
+    .then(response => console.log(response))
```

Comment:
please remove this callback, as we're not using it. Also, the test coverage is dropping/checks are failing cause it says we're not testing this line